fiblary3/tests/unit/test_restapi.py

```diff
@@ -15,10 +15,11 @@
 
 """Test rest module"""
 
+import base64
 import json
-import mock
 
 import requests
+from requests_mock.contrib import fixture
```

Comment:
Personally I would lean towards using pytest fixtures and test functions rather than class-based test cases via the testtools TestCase API, but I wanted to minimise the changes.